lib/transport/selenium-webdriver/method-mappings.js

```diff
@@ -587,13 +587,9 @@ module.exports = class MethodMappings {
         async sendKeysToElement(webElementOrId, value) {
           if (Array.isArray(value)) {
             if (value.includes(undefined)){
-              throw new Error('array args can\'t be undefined');
+              throw new Error('sendKeys - each key must be a number of string');
```

Comment:
it should be `each key must be a number or string`

Reply:
@swrdfish Yes I noticed the same thing but I make the error alike https://github.com/SeleniumHQ/selenium/blob/trunk/javascript/node/selenium-webdriver/lib/webdriver.js#L2521-L2561

Reply:
They have a typo does not mean we should have a typo as well. 

You can also send a PR to selenium-webdriver to fix their typo.

Reply:
Yes I agree with that. I am raising a PR for Selenium and we have to wait until that PR is merged otherwise our testcases will fail.

Reply:
PR : https://github.com/SeleniumHQ/selenium/pull/10487

Reply:
> Yes I agree with that. I am raising a PR to selenium-webdriver and we have to wait until that PR is merged otherwise our testcases will fail.

why should our test cases fail, write the tests in such a way that they accommodate the differences.